packages/frontend/src/view/components/page/Navbar.tsx

```diff
@@ -31,7 +31,7 @@ export function Navbar({ user, searchBar = true, instanceName }: NavbarProps) {
       </a>
       <div className="flex flex-wrap gap-y-2 gap-x-4">
         {searchBar && (
-          <SearchBar className="hidden w-auto min-w-[400px] lg:flex" />
+          <SearchBar tradingMode={instanceName === 'Myria' ? 'spot': 'perpetual'} className="hidden w-auto min-w-[400px] lg:flex" />
```

Comment:
This was consulted with @adamiak

Reply:
@adamiak @Wendrowiec13 the trading mode is already part of backend config and already being passed in various places. Why do we not want to pass it here? Is this because we'd have to pass it everywhere? I think leaving this line of code is going to cause a bug in the future.

Reply:
Exactly. For sure it's going to cause a bug in the future but I would get rid of it together with getInstanceName in a separate PR. What do you think?

Reply:
When I suggested to use "the same solution as we do for the logo" I actually thought that we have access to the trading mode. Sorry for the confusion. Comparing to 'Myria' in this place is just asking for issues in the future. 

Since this is the same situation as with logo, .i.e. we would need to pass some data in every single controller, I suggest we use the same solution that we use for logo - looking at process.env. Although it's not great and we will need to find a better solution in the future. 

But for now, create a function like `getInstanceName()` that will return correct `TradingMode` or throw if unknown value is in the env.

Reply:
Done.